src/viewer.js

```diff
@@ -3100,12 +3100,20 @@ function onCanvasContextMenu( event ) {
     event.preventDefault = eventArgs.preventDefault;
 }
 
+
+/**
+ * Handles the keyup event on the viewer's canvas element.
```

Comment:
Thank you for adding this comment. Please also add a `@private` line so the documentation builder knows this is private.